### PR TITLE
N34: lightingtalk 

### PR DESCRIPTION
**Pull request by alanpoon**
_Part of #1033_

### CODE REVIEW

**Comment by ozkriff** [comment]
:construction: wip

File: content/news/034/index.md [hunk 1]
```diff
@@ -112,6 +112,31 @@ He will be releasing videos about UI in Bevy and Rapier physics next month.
...
+This month [@rustropy_gaming][alanpoon_twitter] joined Cosmonic to give
+a series of Wasmcloud talks in Kubecon 2022 VLC:
+[YouTube recording][Lightning-Talk-youtube], [slides][Lightning-Talk-pdf].
+
+Bevy can be added into Wasmcloud's actor as serverless game server.
+
+Check out Twitter [@rustropy_gaming][alanpoon_twitter]
+for announcement of QQparty going live.
+
+Deploying multiplayer game servers may be cheap in the future.
```
Comment (17cupsofcoffee) [inline content/news/034/index.md:131]: Some suggested tweaks to make the text flow better:

```suggestion
This month, [@rustropy_gaming][alanpoon_twitter] joined Cosmonic to give
a series of Wasmcloud talks in Kubecon 2022 VLC. They demonstrated
how Bevy can be added into Wasmcloud's actors as a serverless
game server.

You can check out a recording of the talk on [YouTube][Lightning-Talk-youtube],
as well as viewing the [slides][Lightning-Talk-pdf]. For updates on QQparty,
follow [@rustropy_gaming][alanpoon_twitter] on Twitter!
```

Comment (ozkriff) [reply content/news/034/index.md:131]: Nice! @alanpoon are you ok with the changes?

Comment (alanpoon) [reply content/news/034/index.md:131]: Yes. Thanks.

**Review by ozkriff — APPROVED** [review]
:rocket:

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**